README.md

```diff
@@ -108,6 +108,21 @@ Kubernetes Slack server.
 If you've spotted a bug, please open an issue
 on our [Github](https://github.com/kong/kubernetes-ingress-controller/issues).
 
+### Preview and Experimental Features
+
+At any time the KIC may include features or options that are considered
+experimental and wont be enabled by default, nor available in the [Kong
```

Comment:
Original was missing apostrophe, may as well also agree in tense with the preceding clause.
```suggestion
experimental and are not enabled by default, nor available in the [Kong
```